cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleServiceTest.java

```diff
@@ -109,6 +119,16 @@ public static void stop() {
     AppFabricTestHelper.shutdown();
   }
 
+  private void setPromotionToProdFlag(boolean promotionToProdFlag) {
```

Comment:
Remove this as we are removing the flag check in the api calls

Reply:
This has been removed in the latest commit. Need to push the changes after the build is verified locally.